gulpfile.js

```diff
@@ -30,3 +30,9 @@ gulp.task('tdd', function(done) {
     configFile: __dirname + '/karma.conf.js'
   }, done);
 });
+
+gulp.task('test', function(done) {
+  karma.start({
+    configFile: __dirname + '/karma-ci.conf.js'
```

Comment:
It would be better to just override configuration properties instead of using a different karma configuration file:

    gulp.task('test', function(done) {
        configFile: __dirname + '/karma.conf.js',
        singleRun: true,
        browsers: ['PhantomJS']
    });